.gitignore

```diff
@@ -26,3 +26,6 @@ node_modules/
 cache
 
 **/*.bin
+
+**.txt**
+**.json**
```

Comment:
This seems like an accident waiting to happen, can you ignore something more specific?

Reply:
Oops, didn't see this.